src/sass/bootstrap-asu.scss

```diff
@@ -125,9 +128,6 @@ $theme-colors: map-remove($theme-colors, 1, 3, 5);
 @import '../../node_modules/bootstrap/scss/utilities';
 @import '../../node_modules/bootstrap/scss/print';
 
-// css that is available after Bootstrap upgrade.
-@import 'bootstrap-asu-upgrade';
-
```

Comment:
We don't want to lose this import which handles our gap-fitting between bootstrap versions.